tuareg-intern.el

```diff
@@ -0,0 +1,119 @@
+;;; tuareg-intern.el --- Helper functions
```

Comment:
Am 21.06.2015 um 23:28 schrieb Christophe Troestler:
>
> In tuareg-intern.el 
> <https://github.com/ocaml/tuareg/pull/54#discussion_r32897882>:
>
> > @@ -0,0 +1,119 @@
> > +;;; tuareg-intern.el --- Helper functions
>
> You put too many things in a single PR. There are tests above, now 
> this is a refactoring of the library. Please split your PR in small 
> easily reviewable chunks.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/ocaml/tuareg/pull/54/files#r32897882>.
>

Hmm, don't know how to do that, as it's a translation from python-mode.el

The API needs some helper functions, which by themselfs wouldn't be that 
impressive and probably never get merged.

Basics is in commit 2ed2643dd6c48d6de8f0e99d55efd4

with *644 additions* and *8 deletions*.

Maybe run the tests to get some impression?